x-pack/plugins/security/public/management/api_keys/api_keys_grid/api_keys_grid_page.tsx

```diff
@@ -194,7 +220,7 @@ export class APIKeysGridPage extends Component<Props, State> {
               ? undefined
               : [
                   <EuiButton
-                    {...reactRouterNavigate(this.props.history, '/create')}
+                    {...reactRouterNavigate(this.props.history, '/flyout')}
```

Comment:
Ooh! Great catch! In the latest commit Ive changed `/create` back to its original flow so as not to make a breaking change.

For update/view, I chose to go with React state, the refresh bug shouldn't be an issue now since the flyout will close automatically on refresh. Please let me know what you think!